test/app-luatest/console_autorequire_test.lua

```diff
@@ -0,0 +1,171 @@
+local t = require('luatest')
+local it = require('test.interactive_tarantool')
+local cbuilder = require('test.config-luatest.cbuilder')
+local cluster = require('test.config-luatest.cluster')
+
+local g = t.group()
+
+local autorequire_list = {
+    'clock',
+    'compat',
+    'config',
+    'datetime',
+    'decimal',
+    'fiber',
+    'fio',
+    'fun',
+    'json',
+    'log',
+    'msgpack',
+    'popen',
+    'uuid',
+    'varbinary',
+    'yaml',
+}
+
+local function assert_no_autorequire(it)
+    it:roundtrip("require('strict').off()")
```

Comment:
Does it mean that user should disable `strict` for the console each time by hands too?

Reply:
I need it to receive the same response for Debug and RelWithDebInfo builds in the test. I don't think that it is a problem for a user of the interactive console.

For example, on Debug build (the strict mode is enabled):

```lua
tarantool> unknown_var
---
- error: '[string "return unknown_var"]:1: variable ''unknown_var'' is not declared'
...
```

It seems OK for me.

Reply:
Got it, thanks for the clarifications!